pkg/services/librarypanels/api.go

```diff
@@ -18,9 +18,12 @@ func (lps *LibraryPanelService) registerAPIEndpoints() {
 
 	lps.RouteRegister.Group("/api/library-panels", func(libraryPanels routing.RouteRegister) {
 		libraryPanels.Post("/", middleware.ReqSignedIn, binding.Bind(createLibraryPanelCommand{}), api.Wrap(lps.createHandler))
+		libraryPanels.Post("/:uid/dashboard/:dashboardId", middleware.ReqSignedIn, api.Wrap(lps.connectHandler))
```

Comment:
I think the dashboards part of the path should be called "dashboards", since its a collection of resources, for example the endpoint /api/library-panels/dashboard/ will returns a collection of dashboards. It's also consistent with /api/library-panels.

```suggestion
		libraryPanels.Post("/:uid/dashboards/:dashboardId", middleware.ReqSignedIn, api.Wrap(lps.connectHandler))
```

Reply:
Fixed